Package.swift

```diff
@@ -10,14 +10,16 @@ let package = Package(
         .library(name: "SourceryJS", targets: ["SourceryJS"]),
     ],
     dependencies: [
-        .package(url: "https://github.com/kylef/Commander.git", from: "0.6.1"),
+        .package(url: "https://github.com/kylef/Commander.git", .exact("0.6.1")),
         // PathKit needs to be exact to avoid a SwiftPM bug where dependency resolution takes a very long time.
         .package(url: "https://github.com/kylef/PathKit.git", .exact("0.8.0")),
-        .package(url: "https://github.com/jpsim/SourceKitten.git", from: "0.18.1"),
-        .package(url: "https://github.com/IBM-Swift/CommonCrypto.git", from: "0.1.5"),
-        .package(url: "https://github.com/SwiftGen/StencilSwiftKit.git", .branch("master")),
+        .package(url: "https://github.com/jpsim/SourceKitten.git", .exact("0.20.0")),
+        .package(url: "https://github.com/IBM-Swift/CommonCrypto.git", .exact("0.1.5")),
+        .package(url: "https://github.com/kylef/Stencil.git", .exact("0.10.1")),
+        .package(url: "https://github.com/SwiftGen/StencilSwiftKit.git", .exact("2.4.0")),
         .package(url: "https://github.com/seanparsons/SwiftTryCatch.git", .revision("e7074a72e4d4dc516f391bc4d4afd8ca6a845b4b")),
```

Comment:
Not sure why this is different in Podfile, but we should probably stop using it at all